crates/configuration/src/native_queries.rs

```diff
@@ -22,13 +22,46 @@ pub struct NativeQuery {
     /// `schema.json`.
     pub result_type: Type,
 
-    /// Arguments for per-query customization
+    /// Arguments to be supplied for each query invocation. These will be substituted into the
+    /// given `command`.
+    ///
+    /// Argument values are standard JSON mapped from GraphQL input types, not Extended JSON.
+    /// Values will be converted to BSON according to the types specified here.
     #[serde(default)]
     pub arguments: BTreeMap<String, ObjectField>,
```

Comment:
Technically this field should be called `parameters`. Is it worth changing?

Reply:
If `parameters` is what we use elsewhere for native queries then it's probably worth changing, given that users will need to write these.

Reply:
ndc-postgres uses the name `arguments` which is what I copied here. It's just that technically they're wrong: parameters are input variables, arguments are the values that bind to parameters. But I can see the argument that consistency is more important than technical correctness.